helpers/actaRecibido/actaRecibidoCrud.go

```diff
@@ -0,0 +1,56 @@
+package actaRecibido
+
+import (
+	"strconv"
+
+	"github.com/astaxie/beego"
+	"github.com/astaxie/beego/logs"
+	"github.com/udistrital/arka_mid/models"
+	"github.com/udistrital/utils_oas/request"
+)
+
+func GetElementoById(id int) (elemento *models.Elemento, outputError map[string]interface{}) {
```

Comment:
Falta documentar, cambiar `...` en el _suggested change_, digamos "GetElementoById hace tal cosa"
```suggestion
// GetElementoById ...
func GetElementoById(id int) (elemento *models.Elemento, outputError map[string]interface{}) {
```